zmsentities/tests/Zmsentities/PermissionTest.php

```
@@ -0,0 +1,20 @@
+/**
+ * @SuppressWarnings(Public)
+ */
+class PermissionTest extends EntityCommonTests
+{
+    public $entityclass = '\BO\Zmsentities\Permission';
+
+    public function testBasic()
+    {
+        $entity = (new $this->entityclass())->getExample();
+        $this->assertEntity($this->entityclass, $entity);
+        $this->assertSame(1, $entity->id);
```

PHPStan (2.1.44)

[error] Call to an undefined method BO\Zmsentities\Tests\PermissionTest::assertSame().


(method.notFound)